front/lib/development.ts

```diff
@@ -17,6 +17,10 @@ export function canForceUserRole(owner: LightWorkspaceType) {
   return isDevelopment() || isADustProdWorkspace(owner);
 }
 
+export function canEnableJIT(owner: LightWorkspaceType) {
```

Comment:
why ? can't we just give it to anyone with the flag ? I want it on my personal workspaces as well 🙏

Reply:
to avoid Extra DB call on other workspaces.

Reply:
mmmh ok

Reply:
but this extra call is coming soon anyway ?

Reply:
We might just skip it and thoroughly test with our production workspace and if no regression is discovered just go full release with it (with a plan to quickly deactivate)

Reply:
But we might also accept to pay the extra query price to roll in 1-2 workspaces to limit the risk but it's nice to not pay that price just to release on our workspace

Reply:
Anyway it's all minor